### PR TITLE
Fix medical doctor's wall locker fill

### PR DESCRIPTION
**Pull request by ChilbroBaggins**
Changelog: (Edited)
- Adds an MD's locker with their gear as a variant of the standing locker.
- Renames original MD's Wall Locker to "Medicine Wall Locker" because it is a variant of the Medicine closet.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by ChilbroBaggins** [comment]
In fact, now that I really think about it, I think that would make more sense. Both would be useful.

**Comment by ChilbroBaggins** [comment]
Failed test doesn't look like it has anything to do with this.

File: Resources/Prototypes/Catalog/Fills/Lockers/medical.yml [hunk 1]
```diff
@@ -70,6 +71,41 @@
...
+        orGroup: Surgshrubs
+      - id: UniformScrubsColorGreen
+        prob: 0.1
+        orGroup: Surgshrubs
+      - id: UniformScrubsColorPurple
+        prob: 0.05
+        orGroup: Surgshrubs
+      - id: NitrousOxideTankFilled
+        prob: 0.3
+      - id: ClothingMaskSterile
```
Comment (AJCM-git) [inline Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:107]: Dont copy paste like this, im pretty sure you can use prototype composition here if you want to inherit both `LockerWallMedical` sprites and `LockerMedicalFilled` fill, also name it properly
```suggestion
- type: entity
  id: LockerWallMedicalFilled
  parent: ["LockerWallMedical", "LockerMedicalFilled"]
```

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/medical.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
 - type: entity
   id: LockerWallMedicalFilled
+  name: Medicine Wall Locker
```
[the remark this answers is not in the document]
Comment (ChilbroBaggins) [reply Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:21]: LockerMedicine is a standup locker. This is a wall locker.

The original issue is that the MD Wall locker contains the Medicine contents not the MD contents.

Unless you mean the NanoMed wall vendor. then I could see we wouldn't need the Medicine wall closet. But the MD locker should still have MD contents.

File: Resources/Prototypes/Catalog/Fills/Lockers/medical.yml (hunk 1 above)
Comment (ChilbroBaggins) [reply Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:107]: I can't change the ID names as they are most likely already mapped on several maps

Comment (ChilbroBaggins) [reply Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:107]: I actually don't know what you want me to do here? I keep looking at it and it's how it's done everywhere else.

Comment (ChilbroBaggins) [reply Resources/Prototypes/Catalog/Fills/Lockers/medical.yml:107]: Oh, sorry, I didn't see that bit at the bottom. of your comment.

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**